scripts/publisher_coverage.py

```diff
@@ -15,7 +15,7 @@
 
 def main() -> None:
     failed: int = 0
-    timeout_in_seconds: int = 10
+    timeout_in_seconds: int = 20
```

Comment:
```suggestion
    timeout_in_seconds: int = 30
```
It seems that I accidentally reduced the timeout to 10 seconds here ca41bb1afc89e792d5b69b5e7897bf311f79c074